src/react-apps/applications/dashboard/src/dashboardServices/createService/createNewService.tsx

```diff
@@ -182,11 +185,22 @@ export class CreateNewServiceComponent extends React.Component<ICreateNewService
           this.showRepoNamePopper(getLanguageFromKey('dashboard.service_name_already_exist', this.props.language));
         } else if (result.repositoryCreatedStatus === 201) {
           window.location.assign(`${altinnWindow.location.origin}/designer/${result.full_name}#/aboutservice`);
+        } else {
+          this.showRepoNamePopper(getLanguageFromKey('dashboard.error_when_creating_service', this.props.language));
+        }
+      }).catch((error: Error) => {
+        console.error('Unsucessful creating new service', error.message);
```

Comment:
Arre you testing this error handling in Jest?

Reply:
Added some now